src/chrome/content/rules/Cooliris.com.xml

```diff
@@ -0,0 +1,14 @@
+<!--
+	Invalid certificate:
+		blog.cooliris.com
+
+-->
+<ruleset name="Cooliris">
+
+	<target host="cooliris.com" />
+	<target host="www.cooliris.com" />
```

Comment:
It's secured in `Bitly_branded_short_domains.xml`. Should I move it here? Or I can also a comment about it.